test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_mnist.py

```diff
@@ -121,11 +153,16 @@ def test_s3_plugin(sagemaker_session, ecr_image, instance_type, region, framewor
                            sagemaker_session=sagemaker_session,
                            image_uri=ecr_image,
                            framework_version=framework_version)
-    estimator.fit('s3://sagemaker-sample-data-{}/tensorflow/mnist'.format(region),
-                  job_name=unique_name_from_base('test-tf-sm-s3-mnist'))
-    _assert_s3_file_exists(region, estimator.model_data)
-    _assert_checkpoint_exists(region, estimator.model_dir, 200)
 
+    inputs = estimator.sagemaker_session.upload_data(
+        path=os.path.join(resource_path, 'mnist', 'data-distributed'),
+        key_prefix='scriptmode/mnist-distributed')
+    estimator.fit(inputs,job_name=unique_name_from_base('test-tf-sm-s3-mnist'))
+    print("=========== Model data location ===============")
```

Comment:
nit: we should configure a logger using python logging library so that we can get things like timestamps and other debug information. This can be done as a followup, since I see we're using `print` in other places in the code, and this is consistent with the existing setup.